.ci/test_process.py

```diff
@@ -0,0 +1,407 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do I need to review this whole file - is it entirely new?
I guess not but I don't see a corresponding deletions...

Reply:
It was a copy of `test_daemon.py` because I initially though I was going to have to change quite a bit to get working and wanted to keep the original working on Travis/Jenkins before dropping those. Turns out it wasn't that big a change. I removed it and just fixed linter issues in `test_daemon.py`.